module/applications/actor/group-sheet.mjs

```diff
@@ -0,0 +1,299 @@
+import ActorMovementConfig from "./movement-config.mjs";
+
+/**
+ * A character sheet for group-type Actors.
+ * The functionality of this sheet is sufficiently different from other Actor types that we extend the base
+ * Foundry VTT ActorSheet instead of the ActorSheet5e abstraction used for character, npc, and vehicle types.
+ */
+export default class GroupActorSheet extends ActorSheet {
+
+  /** @inheritDoc */
+  static get defaultOptions() {
+    return foundry.utils.mergeObject(super.defaultOptions, {
+      classes: ["dnd5e", "sheet", "actor", "group"],
+      template: "systems/dnd5e/templates/actors/group-sheet.hbs",
+      tabs: [{navSelector: ".tabs", contentSelector: ".sheet-body", initial: "members"}],
+      width: 620,
+      height: 620
+    });
+  }
+
```

Comment:
It's awkward. Inheriting from ActorSheet5e would require overriding and removing almost all methods, but it does miss some things that otherwise we do want. It's an example that we probably need to re-rationalize what logic is in the base actor sheet and what logic is in type-specific sheets.